spring-boot-samples/spring-boot-sample-data-geode/.gitignore

```diff
@@ -0,0 +1 @@
+apachegeode-maven-dependency-exclusions.txt
```

Comment:
No, it was/is for my own personal reference (i.e. not absolutely required) and technically [comes from the work](https://github.com/jxblum/spring-session/blob/apache-geode/spring-session-data-geode/build.gradle#L11-L19) on _Spring Session_ to include support for _Apache Geode_ through _Spring Data GemFire_, which has not been integrated yet.

Also, the _GemFire_ team, at my direction (see [GEODE-27](https://issues.apache.org/jira/browse/GEODE-27) for further details), has been busy cleaning up the mess that is _Apache Geode's_ dependencies, so these "exclusions" should not be necessary in the future.

The exclusions were only necessary in _Spring Session_ since they pull in and combine conflicting web related dependencies (e.g. `javax.servlet-api`), which was caused by _Apache Geode's_ unnecessary and incomplete dependency on _Apache HBase_, which pulls in Jetty, which pulls in javax.servlet, and so on, ugh.